spring-cloud-gcp-data-spanner/README.adoc

```diff
@@ -0,0 +1,13 @@
+= Spring Cloud GCP Data Spanner
```

Comment:
We have a separate ref doc for the modules, and here's the  one for Spanner : https://github.com/spring-cloud/spring-cloud-gcp/blob/master/spring-cloud-gcp-docs/src/main/asciidoc/spanner.adoc

Perhaps mention the new transactional annotation in the Transaction sections.

Reply:
Good catch! I didn't even realize it wasn't in the refdoc!